lisp/init-terraform.el

```diff
@@ -0,0 +1,9 @@
+;;; Terraform
+
+(maybe-require-package 'terraform-mode)
+(maybe-require-package 'company-terraform)
+
+(after-load 'terraform-mode
```

Comment:
Same here: this should be conditional upon the `company-terraform` package being available.